src/material/list/_list-theme.scss

```diff
@@ -96,6 +98,8 @@
     // Add values for MDC list tokens.
     @include sass-utils.current-selector-or-root() {
       @include mdc-list-theme.theme($mdc-list-density-tokens);
+      @include token-utils.create-token-values(
```

Comment:
please also add the color, typography, etc. tokens to their respective mixins, so that those work if people add those types of tokens

Reply:
should also make calls to use color and typography tokens even though there aren't any defined yet.